projects/indicators/src/modules/text-highlight/text-highlight.directive.ts

```diff
@@ -30,12 +30,27 @@ export class SkyTextHighlightDirective
    * Specifies the text to highlight.
    */
   @Input()
-  public skyHighlight: string;
+  public set skyHighlight(value: string | string[]) {
+    value = value || [];
+    this._skyHighlight = [];
+    if (typeof value === 'string') {
+      this._skyHighlight = [value as string];
+    } else {
+      for (let index = 0; index < value.length; index++) {
```

Comment:
It's pretty common to use `i` for loop index variable, so I'd switch to that over `index`.

Reply:
Done.